core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -1449,6 +1423,112 @@ export class LoggedOutUser extends BaseUser {
       showMessage(`Activity with title ${expectedActivity} found as expected.`);
     }
   }
+
+  /**
+   * Function to click the Volunteer with Oppia on the about page
+   * and check if it opens the Volunteer form.
+   */
+  async clickVolunteerWithOppiaButtonInAboutPage(): Promise<void> {
+    // The Google Form URL changes from the 1st to the 2nd and from 2nd to the
+    // 3rd in a short span of 500-1000 ms for it's own reasons which we can't
+    // control.So we need to check for all the 3 URLs in the 'allowedVolunteerFormUrls' array
+    // as all of them are valid.
+    const volunteerWithOppiaButtonInAboutPage = this.isViewportAtMobileWidth()
+      ? volunteerWithOppiaMobileButtonInAboutPage
+      : volunteerWithOppiaDesktopButtonInAboutPage;
+    await this.clickLinkButtonToNewTabAndVerifyAllowedUrls(
+      volunteerWithOppiaButtonInAboutPage,
+      'Apply To Volunteer at the top of the Volunteer page',
+      allowedVolunteerFormUrls,
+      'Volunteer Form'
+    );
+  }
+
+  /**
+   * Function to click the Partner With Us button in the About page
+   * and check if it opens the Partnerships Google form.
+   */
+  async clickPartnerWithUsButtonInAboutPage(): Promise<void> {
+    const allowedUrls = [
+      partnershipsFormShortUrl,
+      partnershipsFormUrl,
+      `${partnershipsFormUrl}?usp=send_form`,
+    ];
+
+    const partnerTab = this.isViewportAtMobileWidth()
+      ? partnerMobileTabInAboutPage
+      : partnerDesktopTabInAboutPage;
+    await this.clickOn(partnerTab);
+
+    const partnerWithUsButtonInAboutPage = this.isViewportAtMobileWidth()
+      ? partnerWithUsMobileButtonInAboutPage
+      : partnerWithUsDesktopButtonInAboutPage;
+
+    // The Google Form URL changes from the 1st to the 2nd and from 2nd to the
+    // 3rd in a short span of 500-1000 ms for it's own reasons which we can't
+    // control.So we need to check for all the 3 URLs as all of them are valid.
+    await this.clickLinkButtonToNewTabAndVerifyAllowedUrls(
+      partnerWithUsButtonInAboutPage,
+      'Partner With Us button at the bottom of the Partnerships page',
+      allowedUrls,
```

Comment:
"allowedPartnershipformUrls"

since we would be having similar arrays for the other forms.